README.md

```diff
@@ -158,19 +158,17 @@ You can deploy this project using: Github Pages
 - Twitter: [@twitterhandle](https://twitter.com/M_Usman_Aslam)
```

Comment:
-  [ ] Under the USAGE section mention how can a user run your project IE clicking live on vs code

Reply:
- [ ] Please either update or remove the FAQ section 
![image](https://user-images.githubusercontent.com/62705947/215590870-811bf1e9-95e5-4202-b8f4-cbff7849bc91.png)

Reply:
- [ ] Please fill these sections, your readme will look more professional 
![image](https://user-images.githubusercontent.com/62705947/215591023-dcc0ab75-3405-4f03-9a78-1277eee93b9d.png)

Reply:
- [ ] Remove this description please